### PR TITLE
i18n(es): Updated `/core-concepts/sharing-state.mdx` and translated `recipes/sharing-state.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `/core-concepts/sharing-state.mdx` and translated `recipes/sharing-state.mdx` based on this commit e5ecc42ba415067a434321c0e64816025505bc89

### CODE REVIEW

File: src/content/docs/es/recipes/sharing-state.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+title: Comparte el estado entre componentes de Astro
+description: Aprende como compartir el estado entre componentes de Astro con Nano Stores.
+i18nReady: true
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+:::tip
+¿Estás usando componentes de framework? Mira [como compartir el estado entre Islands](/es/core-concepts/sharing-state/)!
```
Comment (Waxer59) [inline src/content/docs/es/recipes/sharing-state.mdx:11]:
```suggestion
¿Estás usando componentes de framework? Mira [como compartir el estado entre Islas](/es/core-concepts/sharing-state/)!
```

File: src/content/docs/es/recipes/sharing-state.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn add nanostores
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+2. Crea una tienda. En este ejemplo, la tienda rastrea si un diálogo está abierto o no:
```
Comment (Waxer59) [inline src/content/docs/es/recipes/sharing-state.mdx:38]: No estoy seguro si se deberia traducir `store` a `tienda` en este caso, ya que creo que tienda pienso que puede confundir. yo lo dejaria como store
```suggestion
2. Crea un store. En este ejemplo, el store rastrea si un diálogo está abierto o no:
```

File: src/content/docs/es/recipes/sharing-state.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+  ```ts title="src/store.js"
+  import { atom } from 'nanostores';
+
+  export const isOpen = atom(false);
+  ```
+
+3. Importa y utiliza la tienda en una etiqueta `<script>` en los componentes que compartirán el estado.
```
Comment (Waxer59) [inline src/content/docs/es/recipes/sharing-state.mdx:46]:
```suggestion
3. Importa y utiliza el store en una etiqueta `<script>` en los componentes que compartirán el estado.
```

File: src/content/docs/es/recipes/sharing-state.mdx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+Los siguientes componentes `Button` y `Dialog`  utilizan el estado compartido `isOpen` para controlar si un `<div>` en particular está oculto o mostrado:
+
+  ```astro title="src/components/Button.astro"
+  <button id="openDialog">Open</button>
+
+  <script>
+    import { isOpen } from '../store.js';
+    
+    // Configura la tienda a true cuando se hace clic en el botón
```
Comment (Waxer59) [inline src/content/docs/es/recipes/sharing-state.mdx:56]:
```suggestion
    // Configura el store a true cuando se hace clic en el botón
```

File: src/content/docs/es/recipes/sharing-state.mdx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      isOpen.set(true);
+    }
+
+    // Agrega un escuchador de eventos al botón
+    document.getElementById('openDialog').addEventListener('click', openDialog);
+  </script>
+  ```
+
+  ```astro title="src/components/Dialog.astro"
+  <div id="dialog" style="display: hidden">Hello world!</div>
```
Comment (Waxer59) [inline src/content/docs/es/recipes/sharing-state.mdx:67]:
```suggestion
  <div id="dialog" style="display: hidden">¡Hola Mundo!</div>
```

File: src/content/docs/es/recipes/sharing-state.mdx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+  <script>
+    import { isOpen } from '../store.js';
+
+    // Escucha los cambios en la tienda y muestra/oculta el diálogo en consecuencia
```
Comment (Waxer59) [inline src/content/docs/es/recipes/sharing-state.mdx:72]:
```suggestion
    // Escucha los cambios en el store y muestra/oculta el diálogo en consecuencia
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59 😄

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**